### PR TITLE
environment command improvements

### PR DESCRIPTION
**Pull request by FarazPatankar**
### Allow switching to existing environments by name
switch@2x

### Fix cursor in environment selector
cursor@2x

### CODE REVIEW

File: cmd/environment.go [hunk 1]
```diff
@@ -21,17 +22,30 @@ func (h *Handler) Environment(ctx context.Context, req *entity.CommandRequest) e
...
+
+		// Look for existing environment with name
+		for _, projectEnvironment := range project.Environments {
+			if name == projectEnvironment.Name {
+				environment = projectEnvironment
+			}
+		}
+
+		if (environment != nil) {
+			fmt.Printf("%s Environment: %s\n", promptui.IconGood, ui.BlueText(environment.Name))
```
Comment (coffee-cup) [inline cmd/environment.go:35]: nit: The promptui stuff should maybe be added to the ui package

File: cmd/environment.go [hunk 2]
```diff
@@ -21,17 +22,30 @@ func (h *Handler) Environment(ctx context.Context, req *entity.CommandRequest) e
...
+		} else {
+			// Create new environment
+			environment, err = h.ctrl.CreateEnvironment(ctx, &entity.CreateEnvironmentRequest{
+				Name:      name,
+				ProjectID: project.Id,
+			})
+			if err != nil {
+				return err
+			}
+			fmt.Println("Created Environment ✅\nEnvironment: ", ui.BlueText(ui.Bold(name).String()))
```
Comment (coffee-cup) [inline cmd/environment.go:45]: Should this check emoji also be the prompui one?

**Review by coffee-cup — APPROVED** [review]

File: cmd/environment.go (hunk 2 above)
Comment (AndreiJirohHaliliDev2006) [reply cmd/environment.go:45]: Probably, but may not work on terminals that don't support emojis, but let's assume it's fine.

File: cmd/environment.go (hunk 1 above)
Comment (FarazPatankar) [reply cmd/environment.go:35]: Will look into this in a separate PR as there seem to be other instances of it being used as well.

**Pull request merged**